docs/source/tutorial.rst

```diff
@@ -23,7 +23,9 @@ Here we are considering Netcdf4 files and so use the kerchunk ``hdf`` module. Su
     from kerchunk.hdf import SingleHdf5ToZarr 
     import fsspec
 
-Using fsspec to create a pythonic filesystem, provides a convenient way to manage file urls.
+Using fsspec to create a pythonic filesystem, provides a convenient way to manage file urls. 
+
+The ``SingleHdf5ToZarr`` method takes both an ``h5f`` file and a ``url`` as input. The ``h5f`` file can either be a binary Python file-like object or a url, in which case it will be opened using ``fsspec`` and ``storage_options``. The ``url`` input is not used to open the file and is intended to allow the user to compute the reference files on data before it is uploaded to its final storage location. Thus the ``url`` input should be the url of the final file destination and not the current location. 
```

Comment:
I have added this section after out conversation yesterday to make it clear the reference files do not need to be regenerated if the source files move. Although I am not sure how well this comes across here?